appraisal.lic

```diff
@@ -39,6 +41,15 @@ class Appraisal
     @equipment_manager.items.each { |item| break unless appraise_item?(item) }
   end
 
+  def train_appraisal_with_bundle
+    case bput("appraise my bundle quick", 'Roundtime', 'Appraise what', 'You cannot appraise')
+    when 'Appraise what', 'You cannot appraise'
+      return
+    else
+      waitrt?
```

Comment:
```
bput('appraise bundle...
waitrt?
```
would accomplish the same thing, theres no need for a case/switch here

Reply:
this too, thanks.